resources/scss/views/ItemList/Components/Filter/item-filter-list.scss

```diff
@@ -1,4 +1,7 @@
 .list-controls {
+    .card-columns {
```

Comment:
Interesting... How about the solution in the story? Didn't it work?

Reply:
Using "card col" classes instead of "card col-md-4" makes all filters to try to fit in one row instead of three per row.